inc/checks/class-plugin-update.php

```diff
@@ -11,19 +11,23 @@ class Plugin_Update extends Plugin {
 
 	public function run() {
 		$plugins = self::get_plugins();
-		$update_count = 0;
+		$outdated_plugin_names = array();
 		foreach( $plugins as $plugin ) {
 			if ( 'available' === $plugin['update'] ) {
-				$update_count++;
+				$outdated_plugin_names[] = $plugin['name'];
 			}
 		}
+		$update_count = count( $outdated_plugin_names );
+		$outdated_plugin_names = implode( $outdated_plugin_names, ', ' );
 
 		if ( 1 === $update_count ) {
 			$this->set_status( 'warning' );
 			$this->set_message( "1 plugin has an update available." );
+			$this->set_recommendation( "Update the {$outdated_plugin_names} plugin." );
```

Comment:
Sorry I didn't communicate this well-enough originally. The idea is that `doctor` would recommend which WP-CLI command to run next (if possible).

In saying this, we probably need two attributes: `recommendation_message` and `recommendation_command`.

Reply:
I see. Should I try to take a stab at a command and extended message for each so that we can see how that plays out?

Reply:
Go for it!